backend/de.metas.acct.base/src/main/java-legacy/org/compiere/acct/Doc_Invoice.java

```diff
@@ -414,6 +415,18 @@ private List<Fact> createFacts_SalesInvoice(final AcctSchema as)
 					.buildAndAdd();
 		}
 
+		if (invoiceBL.isApply5CentCashRounding(getInvoiceId()))
```

Comment:
you don't have to check it for isApply5CentCashRounding.
Simply just check if cashRoundingAmt != 0 and book it in that case.